### PR TITLE
[stable/fairwinds-insights] FWI-2415 - Add resources recommendations job to charts

### PR DESCRIPTION
**Pull request by jdesouza**
**Why This PR?**
_a short description of why this PR is needed_

Fixes #

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

[Internal Ticket FWI-2415](https://fairwinds.myjetbrains.com/youtrack/issue/FWI-2415)

### CODE REVIEW

File: stable/fairwinds-insights/ci/test-values.yaml [hunk 1]
```diff
@@ -74,6 +74,10 @@ databaseCleanupCronjob:
   enabled: true
   schedules: []
 
+resourcesRecommendationsCronjob:
+  enabled: false
+  schedules: []
```
Comment (rbren) [inline stable/fairwinds-insights/ci/test-values.yaml:79]: I think we should try and enable it during testing, so we catch any issues with the YAML

Comment (rbren) [reply stable/fairwinds-insights/ci/test-values.yaml:79]: Ditto for other cronjobs, but we don't have to add that now

File: stable/fairwinds-insights/values.yaml [hunk 2]
```diff
@@ -351,6 +351,26 @@ databaseCleanupCronjob:
...
+  # -- Enable resources recommendations true by default
+  enabled: true
+  # -- Resources for the resources recommendations job.
+  resources:
+    limits:
+      cpu: 500m
+      memory: 1024Mi
+    requests:
+      cpu: 80m
+      memory: 128Mi
```
Comment (rbren) [inline stable/fairwinds-insights/values.yaml]: these should probably be equal since this is a prod workload

**Review by rbren — APPROVED** [review]

File: stable/fairwinds-insights/ci/test-values.yaml (hunk 1 above)
Comment (jdesouza) [reply stable/fairwinds-insights/ci/test-values.yaml:79]: oh that's right

File: stable/fairwinds-insights/values.yaml (hunk 2 above)
Comment (jdesouza) [reply stable/fairwinds-insights/values.yaml]: You mean limits = requests?

**Pull request merged**